src/sentry/static/sentry/app/components/compactIssue.jsx

```diff
@@ -63,9 +62,14 @@ class CompactIssueHeader extends React.Component {
   };
 
   render() {
-    let {orgId, projectId, data} = this.props;
+    let {data, params: {orgId, projectId}} = this.props;
```

Comment:
Is `params` a globally accepted prop type? I think this was a bit more clear with `orgId` and `projectId` as explicit props.

Reply:
It's part of react router so yes and we use it quite a bit (the component is wrapped with `withRouter`). `projectId` as a prop is a bit confusing though because although the issue might have a project ID that doesn't determine whether it needs to use the new routing or the old one. This should be temporary code until we remove all the old views.